specs/experimental/fault-proof/bond-incentives.md

```diff
@@ -0,0 +1,31 @@
+# Bond Incentives
+
+Bonds is an add-on to the core [Fault Dispute Game](./fault-dispute-game.md). The core game mechanics are
+designed to ensure honesty as the best response to winning subgames. By introducing financial incentives,
+Bonds makes it worthwhile for honest challengers to participate.
+
+- Moves
+- Subgame Resolution
+
+## Moves
```

Comment:
How is a bond sent? Which contract is it sent to?

Reply:
Intentionally unspecified. But the answer is the FDG the bond.

These specs describe the FDG at a high level. There are a couple cases where it aids user understanding to get into the details of contract implementation. But I don't think this is one of them. A different implementation of the FDG may decide to use a separate contract to store bonds.